.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E231,E203,W503,Q000,WPS111,WPS305,WPS348,WPS602,D400,DAR201,S101,DAR101,C812,D104,I001,WPS306,WPS214,D401,WPS229,WPS420,WPS230,WPS414,WPS114,WPS226,WPS442,C819,WPS601,T001,RST304,WPS410,WPS428,A003,A002,I003,WPS221,WPS326,WPS201,S405,DAR301,WPS210,WPS202,WPS213,WPS301,P103,WPS407,WPS432,WPS211,S314,S310,S001,IF100,PT001,PT019
+ignore = E231,E203,W503,Q000,WPS111,WPS305,WPS348,WPS602,D400,DAR201,S101,DAR101,C812,D104,I001,WPS306,WPS214,D401,WPS229,WPS420,WPS230,WPS414,WPS114,WPS226,WPS442,C819,WPS601,T001,RST304,WPS410,WPS428,A003,A002,I003,WPS221,WPS326,WPS201,S405,DAR301,WPS210,WPS202,WPS213,WPS301,P103,WPS407,WPS432,WPS211,S314,S310,S001,IF100,PT001,PT019,I005
```

Comment:
This addition is no longer necessary, because the `flake8-isort` package is no longer part of the build process, so this error will no longer occur.

Reply:
Worth noting that the `I005` flake8 warning is coming from `flake8-isort`, and has to do with import sorting. Isort is opinionated about import ordering, but there are a few notable cases in pytube where import order cannot be alphabetical, so using `isort` is counterintuitive.

Reply:
Ah thanks for clarifying that bit about how isort works! I've reverted the ignore list to its previous state